### ISSUE TITLE
Can we show pinv does NOT generalize?

### ISSUE THREAD

**Issue by brando90** [brando90/overparametrized_experiments-#11]
-does it fall as an example of ERM?
-I assume ERM of logistic regression doesn't count cuz it converges to hard-SVM...but perhaps it DOES overfit since we see that in the LOSS. Would that happen for pinv cuz we are training on the square loss (same metric we train/test)?
